02-electric/electric-hero/electric-hero.css

```diff
@@ -29,19 +30,41 @@
   background-color: var(--level2-background-color);
 }
 
-.electric-hero-img-container{
+@media (min-width:500px){
+  .electric-hero-subcontainer{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+      "electric-hero-image-container electric-hero-copy-container";
```

Comment:
looking at figma design, the image-container is right and the copy-container left.
so re-arrange these.

Reply:
done